### ISSUE TITLE
Question about LoRA tuning using multi-GPU

### ISSUE THREAD

**Issue by guxm2021** [rhymes-ai/Aria#91]
Hi, thank you for open-sourcing your code. I am trying to use LoRA to fine-tune the model under the multi-GPU setup. Here is my command on my customised dataset, the lora config is almost the same as nvr2 example: 

``
accelerate launch --config_file recipes/accelerate_configs/zero3_offload.yaml aria/train.py --config examples/<customized_data>/config_lora.yaml --max_image_size 980
``

The model could be normally fine-tuned and the training loss on my data is very low. When I evaluate the model, I found the adapter_model.safetensors contains no content. So I follow the full fine-tuning setup to transform the checkpoints into adapter_model.bin file. During the evaluation, the lora model still has very high loss. It seems that lora models are not correctly saved or loaded. Could you please clarify my question?
